src/ui/summary-ui-handler.ts

```diff
@@ -345,6 +373,15 @@ export default class SummaryUiHandler extends UiHandler {
 
     this.candyShadow.setCrop(0, 0, 16, candyCropY);
 
+    if (this.friendshipShadow.visible) {
+      this.friendshipShadow.on("pointerover", () => (this.scene as BattleScene).ui.showTooltip("", `${i18next.t("pokemonSummary:friendship")}`, true));
+      this.friendshipShadow.on("pointerout", () => (this.scene as BattleScene).ui.hideTooltip());
+    }
+
+    this.friendshipText.setText(` ${this.pokemon?.friendship||"0"}/255`);
```

Comment:
nit: formatting ( I should really update my pr :pokedollAh: )
```suggestion
    this.friendshipText.setText(`${this.pokemon?.friendship || "0"} / 255`);
```